lte/gateway/c/core/oai/tasks/nas5g/src/M5GAuthenticationFailure.cpp

```diff
@@ -25,6 +25,7 @@ int AuthenticationFailureMsg::DecodeAuthenticationFailureMsg(
   CHECK_PDU_POINTER_AND_LENGTH_DECODER(
       buffer, AUTHENTICATION_FAILURE_MINIMUM_LENGTH, len);
 
+  OAILOG_DEBUG(LOG_NAS5G, "Decoding AuthenticationFailure message");
```

Comment:
AmfMsgDecodeMsg already prints the message type. So we can skip it here.

Reply:
In AmfMsgDecodeMsg() we can make of use of get_message_type_str() to print the message type.